### DISCUSSION TITLE
Use `WeblateOrg/weblate` as a replacement for `translate/pootle` on the ASF Translation Service

### DISCUSSION THREAD

**Discussion by linghengqian** [apache/infrastructure-ideas#9] [Enhancements]
- On https://github.com/translate/pootle/pull/6926 and https://github.com/translate/pootle/issues/6920, the Pootle community reached a consensus that Pootle actually lacks a maintainer, and a possible replacement was mentioned in Pootle's README, which is `WeblateOrg/weblate`.
- Maybe we can consider using `WeblateOrg/weblate` to replace `translate/pootle` on ASF Translation Service https://translate.apache.org/ ? `translate/pootle` is actually just a Web UI for https://github.com/translate/translate . 
- https://github.com/WeblateOrg/weblate can also be used as a Web UI for https://github.com/translate/translate .
- From my perspective, only Apache OpenOffice is using ASF Translation Service.
- ![image](https://github.com/user-attachments/assets/88bc271e-6b84-4cd5-856c-19ab2275df57)

- I noticed https://cwiki.apache.org/confluence/x/vbOoAQ mentions that Infra started testing alternative translation schemes in 2020. What was the end result?

**Comment by Humbedooh** [comment]
We did indeed test out alternatives, but in the end, it wasn't feasible for us due to the added maintenance cost compared with the low overall usage of the translation service. I think bringing the idea of weblate to the openoffice project would be the best path forward right now, as they are currently the sole maintainers of pootle at the ASF.
